.github/build-php-fpm-image.sh

```diff
@@ -0,0 +1,11 @@
+docker image build \
```

Comment:
If there is a shebang on the first line you would not need to run it by `sh .github/build-php-fpm-image.sh`. 

The same applies for `build-elasticsearch-image.sh`